Maple2.Server.Game/Model/Field/Entity/FieldSkill.cs

```diff
@@ -65,12 +74,6 @@ public override void Update(long tickCount) {
             return;
         }
 
-        if (FireCount == 0) {
```

Comment:
Don't we need this? Otherwise the skill will stay in the field indefinitely.

Reply:
it should be handled in Enabled check

Reply:
Enabled is calculate everytime, it checks firecount